include/lib.h

```diff
@@ -6,12 +6,22 @@
 
 #include <types.h>
 
+#ifdef NODEBUG
```

Comment:
With the `#ifdef NODEBUG` and definitions I do get a compiler warning in the apps when running without debug.

```
random-generator/main.c:37:11: warning: unused variable 'stack' [-Wunused-variable]
        uint32_t stack;
```

Stack is used by qemu prints

```
	qemu_puts("Hello, I'm randomapp! &stack is on: ");
	qemu_putinthex((uint32_t)&stack);
	qemu_lf();
```